sycl/doc/developer/DockerBKMs.md

```diff
@@ -37,12 +37,16 @@ identical for Docker and Podman. Choose whatever is available on your system.
 The following containers are publicly available for DPC++ compiler development:
 
 - `ghcr.io/intel/llvm/ubuntu2204_base`: contains basic environment setup for
-   building DPC++ compiler from source.
+   building SYCL compiler from source.
 - `ghcr.io/intel/llvm/ubuntu2204_intel_drivers`: contains everything from the
    base container + pre-installed Intel drivers.
+- `ghcr.io/intel/llvm/ubuntu2204_build`: has development kits installed for
+   Intel/NVidia/AMD and can be used for building SYCL compiler from source with
```

Comment:
```suggestion
   Intel/NVidia/AMD and can be used for building DPC++ compiler from source with
```
I'm not sure if it's fair to say that this image has development kit installed for "Intel". Technically, "development kit" (i.e. L0/Unified Runtime headers and library) are fetched by CMake during "building DPC++ compiler".
Another significant difference is that development kits for NVIDIA and AMD GPUs include drivers, so this image can be used for testing the compiler in addition to the compiler build.